packages/documentation/src/content/docs/concepts/accounting/liquidity.md

```diff
@@ -189,7 +223,7 @@ where
 ```json
 {
   "input": {
-    "eventId": "b4f85d5c-652d-472d-873c-4ba2a5e39052",
+    "incomingPaymentId": "b4f85d5c-652d-472d-873c-4ba2a5e39052",
     "idempotencyKey": "a09b730d-8610-4fda-98fa-ec7acb19c775"
```

Comment:
That's a good idea. I imagine a separate doc would be better since its not liquidity specific. Kinda global to the graphql scope. Then we could reference/link to that doc here. I'll see what make sense.

Reply:
You have

> For Rafiki's GraphQL API, whenever a mutation with an idempotency key is called, the request payload the request response are saved under that key. 

Should that be "the request payload **AND** the request response"? Just want to make sure its not one or the other. I know the payload part is accurate but wasn't sure about response.